console/src/App/App.js

```diff
@@ -259,6 +261,10 @@ const AppState = hoc((configHoc, Wrapped) => {
 			this.setState(({showWelcomePopup}) => ({showWelcomePopup: !showWelcomePopup}));
 		};
 
+		onReset = () => {
+			this.setState({index: 0, welcomeIndex: 0, showWelcomePopup: true, showUserSelectionPopup: false});
```

Comment:
I added resetting `index` to 0 here as well in case the user is on another view when they reset so the "background" view of the app is cleared up.